app/Http/Middleware/CheckUser.php

```diff
@@ -17,18 +17,16 @@ class checkUser
      */
     public function handle($request, Closure $next)
     {
-        if(Auth::check())
-        {   
-            $edit_users = User::findOrFail( $request->route()->parameters('id') );
+        if(Auth::check()) {   
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.SuperfluousWhitespace.EndLine</br>**Severity**: error</br>**File**: app/Http/Middleware/CheckUser.php</br>Whitespace found at end of line